cfme/tests/openstack/test_host_configuration.py

```diff
@@ -16,7 +16,7 @@
 @pytest.mark.usefixtures("setup_provider_modscope")
 def test_host_configuration(provider, soft_assert):
     provider.load_details()
-    sel.click(InfoBlock.element("Relationships", "Nodes"))
+    sel.click(InfoBlock.element("Relationships", "Hosts / Nodes"))
```

Comment:
If this only changed in CF4.2 (CFME 5.7) this needs to be version picked, not static.